agents/kubevirt/fence_kubevirt.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python -tt
+
+import os
+import sys, re
+import logging
+import atexit
+sys.path.append("/usr/share/fence/")
+from fencing import *
```

Comment:
I don't know about the code standards here in fence_agent, but as a reviewer, `import *` makes things quite difficult to me. Can you use only specific imports like in the line below?

Reply:
We usually use * here, and it's a predefined subset as you can see in the fencing library.